### PR TITLE
Respect environment for page_config.json

### PR DESCRIPTION
**Pull request by afshin**
Respects Python environment when handling `page_config.json`

cf. https://github.com/jupyterlab/jupyterlab/issues/9240

### CODE REVIEW

File: jupyterlab_server/config.py [hunk 1]
```diff
@@ -242,3 +249,24 @@ def _default_translations_api_url(self):
...
+    Returns the string to the environment
+    Parameters
+    ----------
+    user : bool [default: False]
+        Get the user's .jupyter config directory
+    sys_prefix : bool [default: False]
+        Get sys.prefix, i.e. ~/.envs/my-env/etc/jupyter
+    """
+    if user and sys_prefix:
+        sys_prefix = False
```
Comment (jasongrout) [inline jupyterlab_server/config.py:265]: if user is true, sys_prefix won't ever get looked at below. Why set sys_prefix to False?

Comment (jasongrout) [reply jupyterlab_server/config.py:265]: Would it be more elegant to just have a level parameter that can be 'user', 'sys_prefix', or 'system'?

File: jupyterlab_server/config.py [hunk 2]
```diff
@@ -53,10 +53,17 @@ def get_federated_extensions(labextensions_path):
     return federated_extensions
 
 
-def get_static_page_config(app_settings_dir=None, logger=None):
+def get_static_page_config(app_settings_dir=None, logger=None, all_levels=False, user=False, sys_prefix=False):
```
Comment (jasongrout) [inline jupyterlab_server/config.py:56]: all_levels, user, and sys_prefix are mutually exclusive here, right? Is it more elegant to have a single parameter that can take three values, with a single default value?

**Review by blink1073 — APPROVED** [review]
Thanks!

**Pull request merged**